lib/tests/fixtures/facts/linux/cloud/azure-unknown

```diff
@@ -0,0 +1 @@
+this has option unknown-245 defined
```

Comment:
I'm not sure what lease files typically look like, but maybe it's worth adding some extra garbage lines here and on the `azure` file just to make sure it can find this option string? Since it looks like the implementation is iterating over each line in the given file.